base/src/main/java/io/spine/code/generate/Generation.java

```diff
@@ -18,7 +18,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.code;
+package io.spine.code.generate;
 
 /**
  * Constants for code generation.
```

Comment:
As long as we repackage. 

Let's move `ModelCompilerAnnotation` to the top level, kill `Generation` (as it isn't what it claims to be) and rename `ModelCompilerAnnotation` to `GeneratedBySpine`. Ideally we need to relate this new class to the `Generated` annotation itself, e.g. return it from this class.